### PR TITLE
doc: add super quick setup steps

### PR DESCRIPTION
**Pull request by bradjc**
### Pull Request Overview

This pull request adds the bare minimum steps you have to do to get Tock working on two common platforms. Every time I'm setting up Tock on a new computer/VM I always want these steps, and I think that experienced developers will appreciate the straight to the point directions. Plus, they don't require you to manually install anything that the build scripts are just going to do for you.

### Documentation Updated

- [x] Kernel: Updated the relevant files in `/docs`, or no updates are required.

### CODE REVIEW

File: doc/Getting_Started.md [hunk 1]
```diff
@@ -11,8 +11,29 @@ developing Tock.
 3. [arm-none-eabi toolchain](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads) (version >= 5.2)
 4. Command line utilities: wget, sed, make, cmake
 
+### Super Quick Setup
+
+MacOS:
+```
+$ curl https://sh.rustup.rs -sSf | sh
+$ brew tap ARMmbed/homebrew-formulae && brew update && brew install arm-none-eabi-gcc
+$ pip3 install tockloader --user
```
Comment (ppannuto) [inline doc/Getting_Started.md]: Do we want to put `--user` here? You need to do extra things to get user pip working out of the box, and brew's installation of pip is a no-sudo, no-user IIRC

Comment (niklasad1) [reply doc/Getting_Started.md]: I agree better with ```sudo``` but I can only speak for ```Linux/ubuntu``` where you must be a member of the ```dialout group``` in order to access to serial port without being ```root```

**Comment by niklasad1** [comment]
I think docker could be a nice alternative to this BTW

File: doc/Getting_Started.md (hunk 1 above)
Comment (bradjc) [reply doc/Getting_Started.md]: I just matched what the tockloader docs say. I'm old school and have never tried --user.

Comment (ppannuto) [reply doc/Getting_Started.md]: Ah, it was breaking it out into OS X and Linux separately here that called more attention to it. I'll push a fix for that now and for @niklasad1's dialout point

**Comment by ppannuto** [comment]
Only concern with a Docker image is that I think it's a bit harder to keep up to date, but I also don't use Docker much / know it well, so that may be unfounded.

**Pull request merged**